### PR TITLE
Basic UI

### PR DESCRIPTION
**Pull request by NalianAchei2020**
> Hello 😁👋, In this PR, I add the UI design with the following requirements 📑
> 
> General requirements 
✅ Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
 ✅ Make sure that you used correct [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md). 
✅ Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).
> 
> Ruby requirements 
✅ Follow our list of [best practices for Ruby](https://github.com/microverseinc/curriculum-ruby/blob/main/articles/ruby_best_practices.md).
> 
> Project requirements Create a app.rb file that will serve as your console app entry-point. It should have methods that do the following: 
✅ List all books. 
✅ List all people.
 ✅ Create a person (teacher or student, not a plain Person). 
✅ Create a book.
 ✅ Create a rental. 
✅ List all rentals for a given person id. In your main.rb define the entry point, this will be a method called main that is invoked at the end of your file. This method should do the following: 
✅ Present the user with a list of options to perform.
 ✅ Lets users choose an option. 
✅ If needed, ask for parameters for the option. 
✅ Have a way to quit the app.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,163 @@
+require_relative 'book'
+require_relative 'teacher'
+require_relative 'student'
```
Comment (shella12) [inline app.rb]: - [ ] Hi @NalianAchei2020, I noticed a typo in the import for the `students` file in `app.rb`. The file name is currently spelled `student`, but it should be spelled `students`. Please fix this typo.

File: main.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+require_relative 'app'
+
+def main
+  app = App.new
+  app.initial
```
Comment (shella12) [inline main.rb]: - [ ] To create a new object in Ruby, you simply need to call the new method on the class. For example, to create a new App object, you would say app = App.new. The initial method is not a valid method on objects in Ruby, so please remove it from your code.

Comment (shella12) [inline main.rb]: - [ ] According to the project requirements you need to create a simple console app in the `main.rb` file that works in the same was as the [example video](https://www.youtube.com/watch?v=vkkgrhD6aXQ) provided in the requirements. Main.rb should be the entry point (hint in this [lesson ](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/console_apps.md#console-app-entry-point)). You can do that by simply calling the `app.display_option` method in the `main.rb` file. Example snippet:

```rb
equire_relative 'app'

def main
  app = App.new;
  app.display_options
end

main

```

File: app.rb [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+      end
+      person_id = gets.chomp.to_i
+      puts 'Enter the date in this format yy/mm/dd: '
+      date = gets.chomp
+      rental = Rental.new(@books[book_id], @people[person_id], date)
+      @rentals.push(rental)
+      puts 'Rental created successfully'
+    end
+  end
+
```
Comment (shella12) [inline app.rb]: - [ ] While trying to create a `rental` I have encountered an error. Please ee the screenshot below and try to fix the error:
Screenshot 2023-10-07 at 9 10 01 PM

**Review by shella12 — CHANGES_REQUESTED** [review]
Hi @NalianAchei2020, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- Added `main.rb` file  ✅ 
- Created console app entry-point `app.rb` file ✅ 
- Created all required methods in `app.rb` ✅ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TuyishimireEric — APPROVED** [review]
# Status: APPROVED 🟢  
### Hi  @NalianAchei2020  👋 👋 

#### I'm @TuyishimireEric, Your code reviewer,
Good job on implementing the project requirements! 🥇 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it 🦾  
Congratulations! 🎉


## Highlight 👍 
✅ Made suggested changes 👍 

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/enus/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**